pkg/webui/console/components/device-data-form/index.js

```diff
@@ -396,13 +358,13 @@ class DeviceDataForm extends Component {
     const joinServerAddress = jsConfig.enabled ? new URL(jsConfig.base_url).hostname : ''
 
     const formValues = {
+      ...emptyValues,
       network_server_address: nsConfig.enabled ? new URL(nsConfig.base_url).hostname : '',
       application_server_address: asConfig.enabled ? new URL(asConfig.base_url).hostname : '',
       join_server_address: external_js ? undefined : joinServerAddress,
-      ...emptyValues,
-      ...initialValues,
       activation_mode: otaa ? 'otaa' : 'abp',
       external_js,
+      may_edit_keys: mayEditKeys,
```

Comment:
```suggestion
      _activation_mode: otaa ? 'otaa' : 'abp',
      _external_js,
      _may_edit_keys: mayEditKeys,
```
I'd prefer that, like we do in the device settings.